packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -674,3 +676,145 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'}];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+function Copyable() {
+  let {clipboardProps} = useClipboard({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'
+      }];
+    }
+  });
+
+  return (
+    <div role="textbox" tabIndex={0} {...clipboardProps}>
+      Copy me 
+    </div>
+  );
+}
+
+export const DropzoneExampleWithInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropZone Area
+      <Input />
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithLinkAndInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Link slot="file">Upload</Link>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithButton = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Button slot="file">upload</Button>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithDraggableObject = (props) => (
+  <div>
+    <Draggable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropZone Area
+    </DropZone>
+  </div>
+);
+
+// copying does not currently work 
+export const DropzoneExampleWithCopyableObject = (props) => (
+  <div>
+    <Copyable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropzoneArea
+    </DropZone>
+  </div>
+);
+
+export const DropzoneWithRenderProps = (props) => (
+  <div>
+    <Draggable />
```

Comment:
similar issue to the above one, if I keyboard drag this, there is no visual indication via the focus ring that focus is on the dropzone, but we do get 'isDropTarget', so maybe we use that instead of the dragover data-attribute I suggested previous and instead have a data-attribute data-is-drop-target or something

I suggest trying them both locally and form an opinion of which one makes more sense to you, then we can discuss which way to go

Reply:
im thinking maybe using isDropTarget since we already define isDropTarget in useDrop as such: "Whether the drop target is currently focused or hovered" which i think fits how we would want this to work